packages/bot-web-ui/src/stores/load-modal-store.ts

```diff
@@ -358,12 +358,17 @@ export default class LoadModalStore implements ILoadModalStore {
 
     onDriveOpen = async () => {
         const { google_drive } = this.root_store;
+        const { verifyGoogleDriveAccessToken } = google_drive;
+        await verifyGoogleDriveAccessToken();
```

Comment:
Shouldn't we return if token expired and user signed out?

Reply:
done ✅